DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -355,7 +350,8 @@ ProviderValueAsAscii (
   )
 {
   EFI_STATUS  Status;
-  CHAR8       *Value = NULL;
+  CHAR8       *Value       = NULL;
+  CHAR8       *AsciiString = NULL;
   UINTN       AsciiSize;
   UINT8       *Buffer;
   BOOLEAN     v = FALSE; // Boolean Types

```

Comment:
Not related to your changes, but a general note, these variable names really need to be cleaned up. `v` and `b` are not helpful at all.

Reply:
Filed #50 to track this.